.github/workflows/velocity.yml

```
@@ -20,25 +20,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.8.2
-      uses: actions/setup-python@v2
+    - uses: actions/setup-python@v2
       with:
         python-version: 3.8.2
-    - name: Install requirements
-      run: pip install -Ur requirements.dev.txt
-      # Install requirements directly instead of going through the
-      # `source environment` -> virtualenv -> `make requirements` chain because
-      # we don't want `source environment` clobbering secrets definition below
-    - name: Initialize environment
+    - name: Check out reports repository
+      uses: actions/checkout@v2
+      with:
+        repository: 'ucsc-cgp/boardwalk-velocity-reports'
+        token: ${{ secrets.azul_velocity_github_token }}
+        path: 'reports/'
+    - name: Set up environment
       run: |
-        mkdir wiki/
-        echo "::set-env name=PYTHONPATH::$(realpath src)"
-        echo "::set-env name=AZUL_DEBUG::0"
+        (cd deployments && ln -snf dev .active)
+        source environment
+        make virtualenv
+        source .venv/bin/activate
+        make requirements
```

To match the Azul readme https://github.com/DataBiosphere/azul#23-project-configuration could this be reordered as:
```suggestion
        make virtualenv
        source .venv/bin/activate
        make requirements
        source environment
```

---

I tested this and it doesn't seem to work because the `requirements` target depends on `check_env`, which will fail if `source environment` hasn't been run. How should I address this? The immediate options to me are to

* Modify `make requirements` to remove the dependency on `check_env`
* Modify README.md to account for the dependency on `check_env`

---

Good catch. Looks like the README is incorrect in this case as the current ordering doesn't work. I vote to change the ordering in the README but it perhaps should go in another ticket, or at very least its own commit.